### PR TITLE
[discovery-ec2] Update the AWS SDK to 1.12.270

### PR DESCRIPTION
**Pull request by arteam**
Resolves https://github.com/aws/aws-sdk-java/security/advisories/GHSA-c28r-hw5m-5gv3

See https://github.com/elastic/elasticsearch/pull/88932

### CODE REVIEW

**Comment by arteam** [comment]
@elasticmachine update branch

File: gradle/verification-metadata.xml [hunk 1]
```diff
@@ -84,11 +84,21 @@
...
             <sha256 value="293cc44d21aac78f5d974992606e7c4bbff20e8d0b897d0f13476424c59df843" origin="Generated by Gradle"/>
          </artifact>
       </component>
+      <component group="com.amazonaws" name="aws-java-sdk-ec2" version="1.12.270">
+         <artifact name="aws-java-sdk-ec2-1.12.270.jar">
+            <sha256 value="faadf443751822e205338e80d2cea5eabd6373c1c3cef6348c24809ca82a9dd0" origin="Generated by Gradle"/>
+         </artifact>
+      </component>
       <component group="com.amazonaws" name="aws-java-sdk-s3" version="1.11.749">
          <artifact name="aws-java-sdk-s3-1.11.749.jar">
```
Comment (tlrx) [inline gradle/verification-metadata.xml:103]: Aren't we suppose to remove these?

**Review by tlrx — APPROVED** [review]
LGTM

**Comment by arteam** [comment]
@elasticmachine update branch

File: gradle/verification-metadata.xml (hunk 1 above)
Comment (arteam) [reply gradle/verification-metadata.xml:103]: Apparently, Gradle doesn't clean them up automatically https://docs.gradle.org/current/userguide/dependency_verification.html#sec:verification-metadata-hygiene

Comment (arteam) [reply gradle/verification-metadata.xml]: I've manually removed the outdated checksums

**Comment by arteam** [comment]
Thanks Tanguy!

**Pull request merged**